bundles/AdminBundle/Resources/public/js/pimcore/settings/thumbnail/item.js

```diff
@@ -369,6 +369,44 @@ pimcore.settings.thumbnail.items = {
         }];
     },
 
+    dragdropRenderer: function (el) {
+        try {
+            // add dnd support
+            let dd = new Ext.dd.DropZone(el.getEl(), {
+                reference: this,
+                ddGroup: "element",
+                getTargetFromEvent: function (e) {
+                    return this.getEl();
+                }.bind(el),
+
+                onNodeOver: function (target, dd, e, data) {
+                    if (data.records.length === 1 && data.records[0].data.elementType === "asset") {
```

Comment:
Maybe we should use 
```suggestion
                    if (data.records.length === 1 && data.records[0].data.type === "image") {
```
here? Because otherwise it is possible to dnd e.g. xlsx files, which doesn´t make sense, I think.

Reply:
Good catch! 👍